homeassistant/components/influxdb/__init__.py

```diff
@@ -331,12 +330,11 @@ class InfluxClient:
 def get_influx_connection(conf, test_write=False, test_read=False):  # noqa: C901
     """Create the correct influx connection for the API version."""
     kwargs = {
-        CONF_TIMEOUT: TIMEOUT,
+        CONF_TIMEOUT: CONF_TIMEOUT,
```

Comment:
I suggest `conf.get(CONF_TIMEOUT, DEFAULT_TIMEOUT)`, unless you make it compulsory.

Reply:
I added this and increased the default to 50.